servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/ReadOnlyClientSecurityConfig.java

```diff
@@ -23,27 +23,26 @@
  * Read-only security config for clients.
  */
 public class ReadOnlyClientSecurityConfig extends ReadOnlySecurityConfig {
-    @Nullable
-    protected String hostnameVerificationAlgorithm = "HTTPS";
-    @Nullable
-    protected String hostNameVerificationHost;
     /**
-     * Only valid if {@link #hostNameVerificationHost} is valid.
+     * See <a href="https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#jssenames">
+     * Endpoint Identification Algorithm Name</a>.
      */
-    protected int hostNameVerificationPort;
+    @Nullable
+    protected String hostnameVerificationAlgorithm = "HTTPS";
```

Comment:
Should this constant be defined somewhere?

Reply:
ideally yes. Ideally this would come from the JDK but [X509TrustManagerImpl](https://github.com/openjdk/jdk11u-dev/blob/master/src/java.base/share/classes/sun/security/ssl/X509TrustManagerImpl.java#L454-L464) just uses the raw string value as well. We could introduce our own constants but we don't really determine what algorithms are and are not supported (we just pass through).